requirements.txt

```diff
@@ -730,12 +730,13 @@ google-resumable-media==2.7.0 \
     --hash=sha256:5f18f5fa9836f4b083162064a1c2c98c17239bfda9ca50ad970ccf905f3e625b \
     --hash=sha256:79543cfe433b63fd81c0844b7803aba1bb8950b47bedf7d980c38fa123937e08
     # via google-cloud-storage
-googleapis-common-protos==1.63.0 \
-    --hash=sha256:17ad01b11d5f1d0171c06d3ba5c04c54474e883b66b949722b4938ee2694ef4e \
-    --hash=sha256:ae45f75702f7c08b541f750854a678bd8f534a1a6bace6afe975f1d0a82d6632
+googleapis-common-protos==1.63.1 \
+    --hash=sha256:0e1c2cdfcbc354b76e4a211a35ea35d6926a835cba1377073c4861db904a1877 \
+    --hash=sha256:c6442f7a0a6b2a80369457d79e6672bb7dcbaab88e0848302497e3ec80780a6a
     # via
     #   google-api-core
     #   grpcio-status
+<<<<<<< HEAD
```

Comment:
@fruch merge conflict was not removed ^

Reply:
not the end of the world... 

this file isn't getting used by anything, it get generate every time we build

it's mainly for tracking

I'll try fixing when I have the chance.

Reply:
PR with fix for it: https://github.com/scylladb/scylla-cluster-tests/pull/7585

Reply:
thanks